lldb/tools/debugserver/source/MacOSX/MachException.cpp

```diff
@@ -18,9 +18,25 @@
 #include "PThreadMutex.h"
 #include "SysSignal.h"
 #include <cerrno>
+#include <inttypes.h>
 #include <sys/ptrace.h>
 #include <sys/types.h>
 
+static void AppendExceptionData(std::vector<mach_exception_data_type_t> &out,
+                                mach_exception_data_t Data,
+                                mach_msg_type_number_t Count) {
+  mach_exception_data_type_t Buf;
+  for (mach_msg_type_number_t i = 0; i < Count; ++i) {
+    // The input Data we receive need not be aligned correctly.
+    // Perform an unaligned copy by pretending we're dealing with
+    // a char* buffer. This is required to work around UBSAN/ASAN
+    // "misaligned address" errors.
+    auto *src = reinterpret_cast<char *>(Data + i);
+    memcpy(&Buf, src, sizeof(mach_exception_data_type_t));
+    out.push_back(Buf);
+  }
+}
```

Comment:
This function is using a combination of LLVM and LLDB code style.
```suggestion
static void AppendExceptionData(std::vector<mach_exception_data_type_t> &out,
                                mach_exception_data_t data,
                                mach_msg_type_number_t count) {
  mach_exception_data_type_t buf;
  for (mach_msg_type_number_t i = 0; i < count; ++i) {
    // The input Data we receive need not be aligned correctly.
    // Perform an unaligned copy by pretending we're dealing with
    // a char* buffer. This is required to work around UBSAN/ASAN
    // "misaligned address" errors.
    auto *src = reinterpret_cast<char *>(data + i);
    memcpy(&buf, src, sizeof(mach_exception_data_type_t));
    out.push_back(buf);
  }
}
```